internal/model/todos.go

```diff
@@ -10,43 +14,42 @@ type Todo struct {
 
 // TodoList kind of a has many struct
 type TodoList struct {
-	ID    string `json:"id"`
-	Name  string `json:"name"`
-	Todos []Todo `json:"todos"`
+	ID    string  `json:"id"`
+	Name  string  `json:"name"`
+	Todos []*Todo `json:"todos"`
 }
 
 // GetTodosListByID returns slice of Todos based on the listID passed in
-func GetTodosListByID(listID string) (TodoList, error) {
+func GetTodosListByID(listID string) (*TodoList, error) {
 	listStatement := `SELECT id, name FROM lists WHERE id = $1`
-	list := TodoList{}
+	list := &TodoList{}
```

Comment:
Common practice to just create the struct as a pointer from the get go. When you have something and you want the address of it (to make the variable a pointer), you use the ampersand.